core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -3852,6 +3853,17 @@ class KafkaApis(val requestChannel: RequestChannel,
         if (exception != null) {
           requestHelper.sendMaybeThrottle(request, consumerGroupDescribeRequest.getErrorResponse(exception))
         } else {
+          if (request.header.apiVersion >= 3 && includeAuthorizedOperations) {
```

Comment:
I suppose that we don't need the version check because this API only has version zero.